src/components/common/navigation/navigation.scss

```diff
@@ -278,13 +278,15 @@
     }
 
     .help-card__option-separator {
-        height: 1px;
+        height: 0px;
```

Comment:
why are we doing this?

Reply:
There was spacing in adding Enterprise support due to this height , so to remove it height it done to 0px.